src/components/tooltip/tooltip.js

```diff
@@ -311,6 +312,7 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $interpolate,
       }
 
       function onElementDestroy() {
+        panelRef && panelRef.destroy();
```

Comment:
You are right about this @ThomasBurleson. It should only be on L300. I will update this asap.